src/GitHub.App/ViewModels/RepositoryCloneViewModel.cs

```diff
@@ -126,9 +130,12 @@ IObservable<Unit> OnCloneRepository(object state)
                     notificationService.ShowError(Resources.RepositoryCloneFailedNoSelectedRepo);
                     return Observable.Return(Unit.Default);
                 }
+                
                 // The following is a noop if the directory already exists.
                 operatingSystem.Directory.CreateDirectory(BaseRepositoryPath);
-                return cloneService.CloneRepository(repository.CloneUrl, repository.Name, BaseRepositoryPath);
+
+                return cloneService.CloneRepository(repository.CloneUrl, repository.Name, BaseRepositoryPath)
+                    .Do(_ => this.usageTracker.IncrementCloneCount());
```

Comment:
Hmm, not sure. I'm tempted to say "the user decided to use the extension to do a clone, so record it - if it didn't work that's a bug rather than a metric"